cl/opinion_page/forms.py

```diff
@@ -215,6 +222,20 @@ class CourtUploadForm(forms.Form):
             }
         ),
     )
+
+    judges = forms.CharField(
+        label="Judges / Lead author",
+        required=False,
+        widget=forms.TextInput(
+            attrs={
+                "class": "form-control",
+                "placeholder": "Judges / Lead author string, use this in case you can't "
```

Comment:
I think you are conflating author/author_str on the opinion object with judges and panel on the cluster.  

```
    author = models.ForeignKey(
        "people_db.Person",
        help_text="The primary author of this opinion as a normalized field",
        related_name="opinions_written",
        on_delete=models.RESTRICT,
        blank=True,
        null=True,
    )
    author_str = models.TextField(
        help_text=(
            "The primary author of this opinion, as a simple text "
            "string. This field is used when normalized judges cannot "
            "be placed into the author field."
        ),
        blank=True,
    )
```

Reply:
So there should be a Charfield field in the form for judges and another for author?